metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
A somewhat better approach to capturing licenses is the npm-license-checker modules, which does what your grep does, but also checks in for LICENSE files and README files with license text in. 

FYI: the license mentioned in see COPYING.txt is tweetnacl, which is Public Domain, so fine.

Apologies for the adhoc analysis, but here is a pivot marked up showing the UNKOWNS and things we need to check. 

The weird one seems to be the LGPL license for xmldom; not sure why this is getting pulled in, but seems to be in the chain angular-cli->ember-cli->testem->xmldom. This seems to be because the old version of angular-cli (as opposed to @angular/cli) brings in the entire ember-cli library, so could be easily avoided.

As for all the unknowns, I've not had chance to go through and check them yet, but thought I'd share this sheet to save others time if they want to go hunting.

[license analysis.xlsx](https://github.com/apache/metron/files/1031647/license.analysis.xlsx)